rfc-126-custom-configuration-for-dependabot.md

```diff
@@ -0,0 +1,246 @@
+# Custom configuration for Dependabot
+
+## Summary
+
+Introduce stricter configuration for Dependabot to reduced number of open PRs, number of deployments and effort required to ensure apps always use the latest version of libraries that we care about.
+
+## Problem
+
+Dependabot was enabled across all our applications in 2018. Due to the number of applications we have and libraries we use, keeping on top of the number of PRs raised by Dependabot has been difficult.
+
+A few solutions have been implemented in the past to make this better:
+
+- [govuk-dependencies] shows the list of open PRs grouped by application, team or gem.
+- [govuk-dependencies] has a Slack bot which automatically messages in each team's channel with a list of open PRs.
+- We [allowed merging Dependabot PRs without requiring two approvals][rfc-103].
+- [Sharing the responsibility for Dependabot PRs across multiple times][dependencies-team].
+
```

Comment:
We've also configured Dependabot to only update top-level dependencies, for many repos.

Reply:
Oh, I actually wasn't aware of this. Do we have some sort of evidence for this I can link to?

Reply:
I don't think so, it's just something I noticed, which isn't all that consistent. I enabled it for Whitehall yesterday on the assumption that it's something we _intended_ to do.

Arguably we're just taking this one step further: all the gems we're proposing to monitor are top-level dependencies. Worth noting this setting hasn't prevented security updates.

Reply:
Added in 4b68907.